openslides/motions/config_variables.py

```diff
@@ -13,7 +13,7 @@ def get_workflow_choices():
     Returns a list of all workflows to be used as choices for the config variable
     'motions_workflow'. Each list item contains the pk and the display name.
     """
-    return [{'value': str(workflow.pk), 'display_name': ugettext_lazy(workflow.name)}
+    return [{'value': str(workflow.pk), 'display_name': workflow.name}
```

Comment:
I am unsure of this change, as it is no string to be translated, but a variable pulled from elsewhere.